datasets/swiss_judgment_prediction/swiss_judgment_prediction.py

```diff
@@ -119,56 +135,70 @@ def _split_generators(self, dl_manager):
             dl_dir = dl_manager.download_and_extract(urls_to_dl)
         except Exception:
             logger.warning(
-                "This dataset is downloaded through Zenodo which is flaky. If this download failed try a few times before reporting an issue"
+                "This dataset is downloaded through Zenodo which is flaky. "
+                "If this download failed try a few times before reporting an issue"
             )
             raise
         return [
             datasets.SplitGenerator(
                 name=datasets.Split.TRAIN,
                 # These kwargs will be passed to _generate_examples
-                gen_kwargs={"filepath": dl_dir["train"], "split": "train"},
+                gen_kwargs={"filepath": dl_dir["train"], "split": "train", "mt_filepath": dl_dir["train_mt"]},
```

Comment:
No need to pass "split" because it is not used in `_generate_examples`:
```suggestion
                gen_kwargs={"filepath": dl_dir["train"], "mt_filepath": dl_dir["train_mt"]},
```